db/migrate/20210806041307_appointments.rb

```diff
@@ -0,0 +1,13 @@
+class Appointments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :appointments do |t|
+      t.date :date
+      t.time :time
+      t.boolean :pay
```

Comment:
minor suggestion here

```suggestion
      t.boolean :pay, null: false, default: false
```
[Avoid the Three-state Boolean Problem](https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem)

Reply:
Thanks, I corrected that on a new commit